tests/test_appliancesmanager.py

```diff
@@ -23,18 +23,23 @@ def assert_appliances_manager_call(
     call_index: int,
     path: str,
     required_headers: dict = None,
+    forbidden_headers: list = None,
 ):
     mock_calls = http_client_mock.mock_calls
 
     call = mock_calls[call_index]
     assert call[0] == "GET"
-    assert call[1].path == path
+    assert path.endswith(call[1].path)
```

Comment:
Can append `path` to `BackendSelectorMock.base_url` here, to keep the comparison stricter as it was?

Reply:
Yep, good call, got that updated